### PR TITLE
Add functionality to fetch message from server and print to portfolio

### PR DESCRIPTION
**Pull request by dhruvi618**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -23,10 +23,10 @@
...
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Dhruvi!</h1>");
-  }
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println("<h1>Hello Dhruvi!!</h1>");
+    }
```
Comment (kmh287) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:31]: Up to you if you want to use +2 or +4 for indentation. Google's style guide recommends 2 and I would expect all the sample code will be written with 2.

Comment (tcwanggoogle) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:31]: tip: I don't know what editor you use, but it's very easy to set indentation by default, and for different file types. Like Kevin said, if the preference for Java is 2, you can set it to 2.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -23,10 +23,10 @@
...
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Dhruvi!</h1>");
-  }
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println("<h1>Hello Dhruvi!!</h1>");
```
Comment (tcwanggoogle) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: nit: Is this added by accident?

Comment (dhruvi618) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: This is what the walkthrough told us to add

**Comment by kmh287** [comment]
Looks like the files in this PR need to be fixed too

**Pull request merged**